sdcm/cluster.py

```diff
@@ -2316,6 +2316,14 @@ def install_mgmt(self, package_url: Optional[str] = None) -> None:
             manager_yaml["tls_key_file"] = tls_key_file
             manager_yaml["prometheus"] = f":{self.parent_cluster.params.get('manager_prometheus_port')}"
 
+            # TODO: remove this restriction to update_frequency configuration after manager-3.3 release
```

Comment:
It must be removed not when 3.3 gets released but when `3.2.6-` stops being supported.
So, need to update this comment.

Reply:
For Manager it's a bit more complicated because we support only latest minor release.
As soon as `3.3.0` version would be released - no support for `3.2.*` but the latest `3.2.*` patch would be used in manager upgrade test: upgrade from `3.2.*` -> `3.3.*`.

Perhaps, it even makes sense to keep this restriction for longer for cases when somebody would want to run tests with Manager < `3.2.7`. So, I'll just delete this `#TODO` block.